runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsBrotherSlainOverlay.java

```diff
@@ -71,14 +73,38 @@ public Dimension render(Graphics2D graphics)
 
 		panelComponent.getChildren().clear();
 
+		final BarrowsBrothers tunnelBrother = plugin.getTunnelBrother();
+
 		for (BarrowsBrothers brother : BarrowsBrothers.values())
 		{
 			final boolean brotherSlain = client.getVar(brother.getKilledVarbit()) > 0;
-			String slain = brotherSlain ? "\u2713" : "\u2717";
+			final boolean isTunnel = brother == tunnelBrother;
+
+			final String icon;
+			final Color color;
+
+			if (brotherSlain)
```

Comment:
Why change the color of the brother's name when there are already icons? That would make the UI inconsistent. There was a reason why the plugin started using icons instead of colors to signify the state.

Reply:
Because you can have a current tunnel and not killed brother? Plus that icon clutters the panel